### ISSUE TITLE
Remove political bias

### ISSUE THREAD

**Issue by byemc** [twitter/the-algorithm#266]
**Is your feature request related to a problem? Please describe.**

I’m always frustrated when random politicians i don’t follow or care about turn up in my feed. 

**Describe the solution you'd like**

Removing political bias from the feed. I’m not saying to remove politicians from the algo, but to not make it biased towards them. 

**Describe alternatives you've considered**

Using the Following feed i guess

**Additional context**

**Comment by nikolqy** [comment]
I agree, I see so many accounts I wouldn't follow in a million years in my feed

**Comment by brandondrew** [comment]
and by "political bias" you mean "bias towards politics" rather than "bias toward a particular political view"?

have you examined the algorithm to determine that there is something (other than the behavior of other users) that makes politicians get more attention than they deserve?

**Comment by lxjv** [comment]
100% agree, i stopped using twitter because of the politics getting thrust into my feed

**Comment by alex-redacted** [comment]
> AND remove NSFW

No, _just_ remove political bias. Let the booties stay. Fly freee.

**Comment by byemc** [comment]
> AND remove NSFW

Maybe enforce content warnings, but if you really think about it removing NSFW would be a bad idea

**Comment by Abastien1734** [comment]
I think there is already a PR for this

**Comment by alex-redacted** [comment]
> The algorithm is serving you content that it deems you might like. Therefore, if you have any political side it will be biased to the side you've shown to like.

**Let me do you one better:** The Twitter algorithm serves you content it thinks you will _engage with_. That doesn't mean you'll like it, it means you'll click, comment on, QRT, and whatnot. Not only that, but if the people you follow engage with content that strikes their emotions? Say, something that upsets them [politically, socially, etc]? You aren't going to escape it, regardless of how many phrases and hashtags you mark as _don't show me this crud_.

I would personally like to see _no_ political crap on my feeds unless it's news and not spun news, either.
